drivers/include/at30tse75x.h

```diff
@@ -0,0 +1,295 @@
+
+/**
+ * @brief   Initialize a AT30TSE75x device
+ *
+ * @param[out] dev          device descriptor
+ * @param[in] i2c           I2C bus the device is connected to
+ * @param[in] speed         I2C speed to use
+ * @param[in] addr          I2C address of the device
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_init(at30tse75x_t* dev, i2c_t i2c, i2c_speed_t speed, uint8_t addr);
+
+/**
+ * @brief   Save configuration register to non-volatile backup register
+ *
+ * @param[in] dev           device descriptor
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_save_config(at30tse75x_t* dev);
+
+/**
+ * @brief   Restore configuration register from non-volatile backup register
+ *
+ * @param[in] dev           device descriptor
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_restore_config(at30tse75x_t* dev);
+
+/**
+ * @brief   Get content of configuration register
+ *
+ * @param[in] dev           device descriptor
+ * @param[out] data         buffer where config register will be written to
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_get_config(at30tse75x_t* dev, uint8_t* data);
+
+/**
+ * @brief   Set content of configuration register
+ *
+ * @param[in] dev           device descriptor
+ * @param[in] data          new value for configuration register
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_set_config(at30tse75x_t* dev, uint8_t data);
+
+/**
+ * @brief   Set temperature resolution
+ *
+ * @param[in] dev           device descriptor
+ * @param[in] resolution    temperature resolution
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_set_resolution(at30tse75x_t* dev, at30tse75x_resolution_t resolution);
+
+/**
+ * @brief   Set operation mode
+ *
+ * @param[in] dev           device descriptor
+ * @param[in] mode          operation mode
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_set_mode(at30tse75x_t* dev, at30tse75x_mode_t mode);
+
+/**
+ * @brief   Set polarity of ALERT pin
+ *
+ * @param[in] dev           device descriptor
+ * @param[in] polarity      polarity of ALERT pin
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_set_alarm_polarity(at30tse75x_t* dev, at30tse75x_alarm_polatity_t polarity);
+
+/**
+ * @brief   Set tolerance to outlying measurements
+ *
+ * @param[in] dev           device descriptor
+ * @param[in] tolerance     tolerance
+ *
+ * @return                  0 on success
+ * @return                  <0 on error
+ */
+int at30tse75x_set_fault_tolerance(at30tse75x_t* dev, at30tse75x_fault_tolerance_t tolerance);
+
+/**
+ * @brief   Set T_Low limit
```

Comment:
I know it's a bit picky but I would have liked to read "Set lower temperature limit" (or similar) here